src/Microsoft.ML.ImageAnalytics/ImageGrayscale.cs

```diff
@@ -216,9 +213,8 @@ protected override Delegate MakeGetter(Row input, int iinfo, Func<int, bool> act
         }
     }
 
-    /// <summary>
-    /// Converts the images to grayscale.
-    /// </summary>
+    /// <include file='doc.xml' path='doc/members/member[@name="ImageGrayscalingEstimator"]/*' />
+    [BestFriend]
```

Comment:
>BestFriend [](start = 5, length = 10)

Why BestFriend, it's still public

Reply:
meant to put it on the ctor; typo. 

---
In reply to: [253242420](https://github.com/dotnet/machinelearning/pull/2372#discussion_r253242420) [](ancestors = 253242420)